packages/client/wallets/smart-wallet/src/types/internal.ts

```diff
@@ -40,7 +40,7 @@ export interface PasskeyCreationContext extends WalletCreationContext {
 }
 
 export interface EOACreationContext extends WalletCreationContext {
```

Comment:
Can rename this stuff later as needed, want to ship this API change quickly before Racino starts migrating and we do the public announcement.